clean-lumber.lic

```diff
@@ -35,23 +35,23 @@ class CleanLumber
         end
         size = right_hand.split.last
       end
-      unless bput('get my saw', 'wood saw', 'You get') == 'wood saw'
-        fput('stow saw')
+      unless /woodcutting saw|wood saw/ =~ bput('get my saw', 'wood saw', 'woodcutting saw', 'You get')
+        fput("put my saw in my #{args.storage}")
         bput('get my second saw', 'wood saw', 'You get')
       end
-      until bput("cut #{size} with my saw", 'roundtime', 'you complete') == 'you complete'
+      until 'you complete' == bput("cut #{size} with my saw", 'roundtime', 'you complete')
         waitrt?
       end
       pause
       waitrt?
-      fput('stow saw')
-      fput('get drawknife')
-      until bput("scrape #{size} with my drawknife", 'roundtime', 'work completes') == 'work completes'
+      fput("put my saw in my #{args.storage}")
+      fput('get my drawknife')
```

Comment:
They can be stored on belts, yes,, but I used the same logic here as with the saw -- that the lumber container would be a more likely storage spot than the default stow container.  I haven't worked with the syntax of using a belt yet, and was just going for a simple change on this one.